.github/workflows/github-release.yaml

```diff
@@ -0,0 +1,22 @@
+name: github-release
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  github-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Release to GitHub
+        run: |
+          gh release create "${{ github.ref_name }}" \
+            --draft \
```

Comment:
You can remove this `--draft`.
Please see https://cli.github.com/manual/gh_release_create for other options.